homeassistant/components/knx/manifest.json

```diff
@@ -2,7 +2,7 @@
   "domain": "knx",
   "name": "KNX",
   "documentation": "https://www.home-assistant.io/integrations/knx",
-  "requirements": ["xknx==0.15.0"],
+  "requirements": ["xknx==0.15.3"],
```

Comment:
this also eliminates a lot of bugs introduces since HA 0.115 and should provide much more stable tunnelling connections.